_data/translations.yml

```diff
@@ -428,3 +320,30 @@ error-back-to-faq:
   nl: Ga naar veelgestelde vragen
   en: Go to frequently asked questions
   
+information-title:
+  nl: Zo gaan we om met jouw gegevens
+  en: Zo gaan we om met jouw gegevens
```

Comment:
Wat is nu het proces rond vertalen naar het engels? Want hiermee introduceren we wel een aantal onvertaalde onderdelen

Reply:
Dit is ondertussen besproken met de content manager/VWS communicatie